app/src/main/java/org/simple/clinic/search/PatientSearchScreenController.kt

```diff
@@ -101,6 +99,7 @@ class PatientSearchScreenController @Inject constructor(
         .andThen(Observable.just { ui: Ui -> ui.openPersonalDetailsEntryScreen() })
   }
 
+  // TODO: This doesn't add any value. Call the function from the UI directly.
```

Comment:
If it's called directly from the UI, the analytics interaction events won't get reported.